app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("All friends should be vaccinated")
+        if visitor["vaccine"].get("expiration_date") < datetime.date.today():

```

Comment:
There is a potential issue in this line. If the `expiration_date` field is not present in the `vaccine` dictionary, this line will raise an AttributeError. Consider adding a check for the existence of the `expiration_date` field before comparing it to the current date.